go/libkb/proof_support_coinbase.go

```diff
@@ -83,6 +84,29 @@ func (t CoinbaseServiceType) CheckUsername(s string) (err error) {
 	return
 }
 
+func coinbaseUserURL(s string) string {
+	return "https://coinbase.com/" + s
+}
+
+func coinbaseSettingsURL(s string) string {
```

Comment:
looking at other functions here, should it be "CoinbaseSettingsURL()" not "coinbaseSettingsURL()" ? Just curious....I don't know what the preferred Go style is...